thentos-tests/tests/Thentos/Backend/Api/PurescriptSpec.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE DataKinds                                #-}
+{-# LANGUAGE ExistentialQuantification                #-}
+{-# LANGUAGE FlexibleContexts                         #-}
+{-# LANGUAGE FlexibleInstances                        #-}
+{-# LANGUAGE GADTs                                    #-}
+{-# LANGUAGE InstanceSigs                             #-}
+{-# LANGUAGE MultiParamTypeClasses                    #-}
+{-# LANGUAGE OverloadedStrings                        #-}
+{-# LANGUAGE RankNTypes                               #-}
+{-# LANGUAGE ScopedTypeVariables                      #-}
+{-# LANGUAGE TupleSections                            #-}
+{-# LANGUAGE TypeOperators                            #-}
+{-# LANGUAGE TypeSynonymInstances                     #-}
+
+module Thentos.Backend.Api.PurescriptSpec (spec, tests)
+where
+
+import Control.Monad.State (liftIO)
+import Data.Configifier ((>>.))
+import Data.Proxy (Proxy(Proxy))
+import Data.String.Conversions (cs)
+import Network.Wai (Application)
+import Network.Wai.Test (simpleBody)
+import Servant.API ((:<|>)((:<|>)), (:>))
```

Comment:
the problem with import wildcards is that you can't search for type constructor imports any more.  it's not a big deal in this case because type name and constructor name are the same, but in general it may be annoying.